CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 
 
+# Upcoming release
+
+ - switch from `:root` to `:not(#\20)` to get more specificity in selectors
+   - Thanks to [@subzey][] for the [idea][] and [@iamstarkov][] for the contribution [#9][]
+
+[idea]: https://twitter.com/subzey/status/829050478721896448
+[@subzey]: https://github.com/subzey
+[@iamstarkov]: https://github.com/iamstarkov
+[#9]: https://github.com/MadLittleMods/postcss-increase-specificity/pull/9

```

Comment:
I prefer inline linking vs references. Seems fine to keep it for now as it will be changed in the release.